package.json

```
@@ -35,7 +32,8 @@
     "snyk": "^1.168.0"
   },
   "peerDependencies": {
-    "n-ui-foundations": "^9.0.0"
+    "n-ui-foundations": "^9.0.0",
+    "@financial-times/n-myft-ui": "^28.0.6"
```

The version should be same as the one in your next-stream-page PR.

```suggestion
    "@financial-times/n-myft-ui": "^28.3.0"
```